docs/setup/settings.asciidoc

```diff
@@ -16,6 +16,8 @@ you'll need to update your `kibana.yml` file. You can also enable SSL and set a
 `elasticsearch.customHeaders:`:: *Default: `{}`* Header names and values to send to Elasticsearch. Any custom headers
 cannot be overwritten by client-side headers, regardless of the `elasticsearch.requestHeadersWhitelist` configuration.
 
+`elasticsearch.logQueries:`:: *Default: `false`* Logs queries sent to Elasticsearch. Requires logging.verbose set to true. This is useful for seeing the query DSL generated by applications that currently do not have a spy panel, for example Timelion and X-Pack Monitoring.
+
 `elasticsearch.pingTimeout:`:: *Default: the value of the `elasticsearch.requestTimeout` setting* Time in milliseconds to
```

Comment:
Use code font for `logging.verbose` and `true`

Replace X-Pack Monitoring with {monitoring}